### PR TITLE
DR 114 Update Next

### PR DESCRIPTION
**Pull request by RubySpeeders**
### Summary of Changes

---
This PR updates Next.js version to the most recent release.

### Summary of Implementation Logic

---

### Issue Link

Original Issue:

Dependent Issues:

Depends on Issues:

---

### Screen shots / Videos

---

### Acceptance Criteria

---

### Checklist for PR Owner

- [ ] Standardized PR Title
- [ ] Included relevant videos and screen shots
- [ ] Included Summary of Changes
- [ ] Included Summary of Implementation Logic
- [ ] Acceptance Criteria are fufilled

### CODE REVIEW

**Comment by RubySpeeders** [comment]
A few thoughts:
I think we can also get rid of the `pages` directory.
I also vote we get rid of the `NavBar` file in the `components` directory, since we don't have one right now; I think it comes standard with Next.js. But I was wondering, should the file names in the `components` directory be capitalised? Just making sure that's not some sort of Next.js config we need to worry about.

File: client/package.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
     "axios": "^1.4.0",
```
Comment (RubySpeeders) [inline client/package.json:19]: I think we can remove this dependency now!

Comment (anquail) [reply client/package.json]: I can't reply to your other comment so responding to everything here!
- `pages` will be deleted when we merge this PR (it just looks strange on GH)
- Just pushed new commits to get rid of `NavBar` and `axios` 
- Next doesn't specify naming conventions, so we are free to do whatever we want! I'm fine to change our component filenames to PascalCase if we want to though!

Comment (RubySpeeders) [reply client/package.json]: Sounds good! I'm fine with the naming convention we have now, just wanted to make sure it wasn't something we needed to keep track of with the new App Router. Thank you for answering all my questions!

Comment (anquail) [reply client/package.json]: Of course! Just chilling here on the bus happy to have something to do

**Review by anquail — APPROVED** [review]

**Pull request merged**